README.md

```diff
@@ -2,35 +2,43 @@
 
 # Fosstars ratings
 
-This action calculates [a security rating for an open source project](https://sap.github.io/fosstars-rating-core/oss_security_rating.html).
-The rating contains a security score and a label. The score is a number from 0 to 10.
-It shows how well the project cares about security. The label can be `Good`, `Moderate`, `Bad` and `Unclear`.
-
-The rating procedure takes into account multiple factors:
-
-*  How well the open source project implements security testing.
-*  How well the project’s community is aware of best security practices and adopt them.
-*  How well vulnerabilities are patched in the project.
-*  How the community commits to supporting the project.
-*  How active and popular the project is.
+This action calculates [a security rating](https://sap.github.io/fosstars-rating-core/oss_security_rating.html) or [an OSS rule of play rating](https://sap.github.io/fosstars-rating-core/oss_rules_of_play_rating.html) for an open source project.
```

Comment:
Could you please add an example with `data-provider-config-urls`?

Reply:
Of course, I will add such an example. However, not sure about the additional sections in the README. In the end it would probably duplicate the description from the main project. Therefore, I tend to keep it that way for now....